src/datastore_export/export_datastore_to_big_query_handler.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+import webapp2
+
+from src.configuration import configuration
+from src.datastore_export.export_datastore_to_gcs_service import \
+    ExportDatastoreToGCSService
+from src.datastore_export.export_gcs_to_big_query_service import \
+    ExportGCSToBigQueryService
+
+
+class ExportDatastoreToBigQueryHandler(webapp2.RequestHandler):
+    def get(self):
+        logging.info("Scheduling export of Datastore entities to GCS ...")
+        output_url = ExportDatastoreToGCSService\
+            .invoke(self.request, self.response)\
+            .wait_till_done(timeout=600)
```

Comment:
could we simply raise an alert via error reporting if the whole request took more than 10 minutes?